.NET/CreateAllPackages.cmd

```diff
@@ -30,7 +30,7 @@ popd
 pushd Microsoft.Recognizers.Text.Choice
 call CreatePackage.cmd
 popd
-pushd Microsoft.Recognizers.DataTypes.DateTime
+pushd Microsoft.Recognizers.Text.DataTypes.TimexExpression
```

Comment:
The new namespace could be Microsoft.Recognizers.Text.DataTypes.TimeExpression, without the X in Timex as it's redundant. But I'm fine with keeping it.

Reply:
should be good now